src/tests/utils/ApiTestSteps.ts

```diff
@@ -546,3 +546,23 @@ function validateCriVcIssuedFailedChecks(txmaEvent: any, yotiMockId: any, vcData
 
 	}
 }
+
+export async function prepareForCallback(docSelectionData:any, sessionId: string): Promise<any> {
```

Comment:
Think the return type can be `Promise<void>`

Reply:
Updated and pushed